includes/Plugin.php

```diff
@@ -218,6 +220,12 @@ public function register() {
 		$activation_notice = new Activation_Notice( $activation_flag );
 		$activation_notice->init();
 
+		$this->integrations['amp'] = new AMP();
+		add_action( 'init', [ $this->integrations['amp'], 'init' ] );
+
+		$this->integrations['jetpack'] = new Jetpack();
+		add_action( 'init', [ $this->integrations['jetpack'], 'init' ] );
+
 		$this->integrations['site-kit'] = new Site_Kit( $this->analytics );
 		add_action( 'init', [ $this->integrations['site-kit'], 'init' ] );
```

Comment:
Can fix this lint error before I approve?

Reply:
@spacedmonkey See discussion in #5141 about this.

This PR is WIP/DRAFT until #5141 is resolved

Reply:
This should fix it https://github.com/google/web-stories-wp/pull/5141/commits/aaebcdce349d24404b692cf212789c3fd0427a24